tikv-client/src/main/java/com/pingcap/tikv/jdbc/AbstractJDBCClient.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2022 PingCAP, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.pingcap.tikv.jdbc;
+
+import com.pingcap.tikv.operation.ErrorHandler;
+import com.pingcap.tikv.operation.JDBCErrorHandler;
+import com.pingcap.tikv.policy.JDBCRetryPolicy;
+import com.pingcap.tikv.util.BackOffer;
+import com.pingcap.tikv.util.ConcreteBackOffer;
+import java.sql.Connection;
+import java.sql.Driver;
+import java.sql.SQLException;
+import java.util.Properties;
+import java.util.concurrent.Callable;
+import lombok.SneakyThrows;
+
+public class AbstractJDBCClient {
```

Comment:
It's weird to name a un abstract class start with AbstarcXXX

Reply:
I will change it to an abstract class.